docker/Dockerfile

```diff
@@ -55,24 +49,19 @@ RUN pip install --prefix="/install" --no-warn-script-location \
 ### Stage 1: runtime
 ###
 
-FROM docker.io/python:${PYTHON_VERSION}-alpine3.10
+FROM docker.io/python:${PYTHON_VERSION}-slim
 
-# xmlsec is required for saml support
-RUN apk add --no-cache --virtual .runtime_deps \
-        libffi \
```

Comment:
we need a bunch of this stuff for features that people expect to work in the docker image.

Reply:
so because we use the available wheels, we don't, because cffi and etc ship what's required.

Reply:
I think we still need xmlsec (or, as debian calls it, `xmlsec1`) at least. pysaml relies on the `/usr/bin/xmlsec1` binary.